### ISSUE TITLE
`npm run serve` failed with an error which is related with gulp

### ISSUE THREAD

**Issue by sembozdemir** [gdg-x/hoverboard#325]
## Expected outcome

Project should be served successfully.

## Actual outcome

It is failed during run command `gulp serve`

## Steps to reproduce

1. Fork the project and clone it locally.
2. `cd` into the base directory of your Hoverboard project.
3. run `npm install`
4. run `npm run serve`
5. Failed.

```
 ...OtherProjects/DevFestIzmir17Hoverboard/hoverboard   master ✔ 
$ npm run serve

> hoverboard@1.2.0 serve /Users/bozdemirs/OtherProjects/DevFestIzmir17Hoverboard/hoverboard
> gulp serve

/Users/bozdemirs/OtherProjects/DevFestIzmir17Hoverboard/hoverboard/node_modules/polymer-analyzer/lib/model/class.js:71
        ({
         ^

ReferenceError: Invalid left-hand side in assignment
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/Users/bozdemirs/OtherProjects/DevFestIzmir17Hoverboard/hoverboard/node_modules/polymer-analyzer/lib/model/model.js:19:10)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)

npm ERR! Darwin 16.6.0
npm ERR! argv "/usr/local/Cellar/node/5.10.1/bin/node" "/usr/local/bin/npm" "run" "serve"
npm ERR! node v5.10.1
npm ERR! npm  v3.8.3
npm ERR! code ELIFECYCLE
npm ERR! hoverboard@1.2.0 serve: `gulp serve`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hoverboard@1.2.0 serve script 'gulp serve'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the hoverboard package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp serve
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs hoverboard
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls hoverboard
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/bozdemirs/OtherProjects/DevFestIzmir17Hoverboard/hoverboard/npm-debug.log
```

And this is npm-debug.log:
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node/5.10.1/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'serve' ]
2 info using npm@3.8.3
3 info using node@v5.10.1
4 verbose run-script [ 'preserve', 'serve', 'postserve' ]
5 info lifecycle hoverboard@1.2.0~preserve: hoverboard@1.2.0
6 silly lifecycle hoverboard@1.2.0~preserve: no script for preserve, continuing
7 info lifecycle hoverboard@1.2.0~serve: hoverboard@1.2.0
8 verbose lifecycle hoverboard@1.2.0~serve: unsafe-perm in lifecycle true
9 verbose lifecycle hoverboard@1.2.0~serve: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:/Users/bozdemirs/OtherProjects/DevFestIzmir17Hoverboard/hoverboard/node_modules/.bin:/usr/local/Cellar/node/5.10.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
10 verbose lifecycle hoverboard@1.2.0~serve: CWD: /Users/bozdemirs/OtherProjects/DevFestIzmir17Hoverboard/hoverboard
11 silly lifecycle hoverboard@1.2.0~serve: Args: [ '-c', 'gulp serve' ]
12 silly lifecycle hoverboard@1.2.0~serve: Returned: code: 1  signal: null
13 info lifecycle hoverboard@1.2.0~serve: Failed to exec serve script
14 verbose stack Error: hoverboard@1.2.0 serve: `gulp serve`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:239:16)
14 verbose stack     at emitTwo (events.js:100:13)
14 verbose stack     at EventEmitter.emit (events.js:185:7)
14 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:24:14)
14 verbose stack     at emitTwo (events.js:100:13)
14 verbose stack     at ChildProcess.emit (events.js:185:7)
14 verbose stack     at maybeClose (internal/child_process.js:850:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
15 verbose pkgid hoverboard@1.2.0
16 verbose cwd /Users/bozdemirs/OtherProjects/DevFestIzmir17Hoverboard/hoverboard
17 error Darwin 16.6.0
18 error argv "/usr/local/Cellar/node/5.10.1/bin/node" "/usr/local/bin/npm" "run" "serve"
19 error node v5.10.1
20 error npm  v3.8.3
21 error code ELIFECYCLE
22 error hoverboard@1.2.0 serve: `gulp serve`
22 error Exit status 1
23 error Failed at the hoverboard@1.2.0 serve script 'gulp serve'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the hoverboard package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     gulp serve
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs hoverboard
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls hoverboard
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]

```

**Comment by ozasadnyy** [comment]
@sembozdemir just tried and works on my machine. Try to update NodeJS and reinstall all dependencies

**Comment by Splaktar** [comment]
@ozasadnyy can you please update the `package.json`?
```
  "engines": {
    "node": ">=4.0"
  },
```
This should now be what? Is only `>=8.1` supported now?

**Comment by ozasadnyy** [comment]
@Splaktar I need confirmation from @sembozdemir that it has fixed the issue

**Comment by sembozdemir** [comment]
Thanks! When I updated node version to v8.6.0, it works properly. Documentation should also be edited.

**Comment by ozasadnyy** [comment]
Fixed in #329
